src/main/scala/com/fulcrumgenomics/fasta/CollectAlternateContigNames.scala

```diff
@@ -124,10 +124,13 @@ class CollectAlternateContigNames
  @arg(flag='a', doc="The assembly report column(s) for the alternate contig name(s)", minElements=1) val alternates: Seq[AssemblyReportColumn],
  @arg(flag='s', doc="Only output sequences with the given sequence roles.  If none given, all sequences will be output.", minElements=0)
   val sequenceRoles: Seq[SequenceRole] = Seq.empty,
+ @arg(flag='d', doc="Add aliases to this existing sequence dictionary file.  The primary contig must match.") val existing: Option[PathToSequenceDictionary] = None,
```

Comment:
Maybe add a sentence about this in the class level usage too?  There's also more room there to explain "The primary contig must match".  I assume that means that _all_ the sequences in the input dictionary must match to a sequence in the assembly report by `--primary` name but that it's ok for the assembly report to have more records?  Might also explain whether or not you match against all aliases on both sides.